src/places/services/state-machine/places.transitions.ts

```diff
@@ -0,0 +1,6 @@
+import { PlaceState } from '../../types/place.state.enum';
+
+export const placesTransitions = [
+  { startState: PlaceState.ACTIVE, endState: PlaceState.INACTIVE, name: 'DEACTIVATE' },
```

Comment:
PLease crate an enum `enum TransitionName { DEACTIVATE = 'DEACTIVATE', ACTIVATE = 'ACTIVATE' }`
and reuse in DTO